test/e2e/service_bootstrap.py

```diff
@@ -61,13 +59,14 @@ def create_vpc() -> str:
     return vpc_id
 
 
-def create_subnet(vpc_id: str) -> str:
+def create_subnet(ec2, vpc_id: str, az: str, cidr=VPC_SUBNET_CIDR_BLOCK[0]) -> str:
```

Comment:
no need to set a default value for the `cidr` parameter since you always specify it...

Reply:
will update